source/docs/software/wpilib-tools/robotbuilder/introduction/robotbuilder-created-code.rst

```diff
@@ -10,41 +12,189 @@ A RobotBuilder generated project consists of a package (in Java) or a folder (in
 
 Autogenerated Code
 ------------------
+.. tabs::
+
+    .. code-tab:: java
+
+        // BEGIN AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=REQUIRES
+        requires(Robot.claw);
+        // END AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=REQUIRES
+        setTimeout(1);
 
-.. image:: images/robotbuilder-created-code-2.png
+    .. code-tab:: cpp
+
+        // BEGIN AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=REQUIRES
+        Requires(Robot::claw);
+        // END AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=REQUIRES
+        SetTimeout(1);
 
 When the robot description is modified and code is re-exported RobotBuilder is designed to not modify any changes you made to the file, thus preserving your code. This makes RobotBuilder a full-lifecycle tool. To know what code is OK to be modified by RobotBuilder, it generates sections that will potentially have to be rewritten delimited with some special comments. These comments are shown in the example above. Don't add any code within these comment blocks, it will be rewritten next time the project is exported from RobotBuilder.
 
 If code inside one of these blocks must be modified, the comments can be removed, but this will prevent further updates from happening later. In the above example, if the //BEGIN and //END comments were removed, then later another required subsystem was added in RobotBuilder, it would not be generated on that next export.
 
 Main Robot Program
 ------------------
+.. tabs::
+
+    .. tab:: Java
+
+        .. code-block:: java
```

Comment:
I think you can use code-tabs here

Reply:
I can't. The code-tab doesn't support line numbers or highlighting.